Src/MainLib/InterceptFrame.cpp

```diff
@@ -903,13 +908,45 @@ bool InterceptFrame::IsBufferReadable(GLenum bufType, uint bufferCount) const
   return false;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+//
+void InterceptFrame::GetBufferViewSize(GLint viewSize[4]) const
+{
+#ifdef GLI_BUILD_WINDOWS
+  if (this->fullscreen)
+  {
+      GLint fboId = 0;
+      GLV.glGetIntegerv(GL_READ_FRAMEBUFFER_BINDING, &fboId);
```

Comment:
Probably should check the extensionFBO flag before calling this

Reply:
Done.